moneta/moneta/legend/legend.py

```diff
@@ -36,21 +37,25 @@ def b(): self.accesses = Accesses(model, self.update_selection)
         def c(): self.stats = PlotStats(model)
         def d(): self.tags = Tags(model, self.update_selection, tag_type=TAG_TYPE_SPACETIME)
         def e(): self.threads = Tags(model, self.update_selection, tag_type=TAG_TYPE_THREAD)
+        def m(): self.mearsurement = PlotMeasure(model)
         def f(): self.add_panel(LEGEND_MEM_ACCESS_TITLE, self.accesses.widgets)
         def g(): self.add_panel(LEGEND_TAGS_TITLE, self.tags.widgets)
         def h(): self.add_panel(LEGEND_THREADS_TITLE, self.threads.widgets)
         def i(): self.add_panel(LEGEND_STATS_TITLE, self.stats.widgets)
+        def n(): self.add_panel('Measurement', self.mearsurement.widgets)
```

Comment:
Please make 'Measurement' a Constant in settings.py

Reply:
Fixed.